scipy/sparse/tests/test_matrix_io.py

```diff
@@ -43,5 +50,35 @@ def test_save_and_load_one_entry():
     _check_save_and_load(dense_matrix)
 
 
+@dec.skipif(NumpyVersion(np.__version__) < '1.10.0', 'feature requires numpy >= 1.10.0')
```

Comment:
It would be nice if the message said what feature we're requiring here, like "disabling unpickling requires numpy >= 1.10.0" (or similar).